internal/pkg/model/state.go

```diff
@@ -13,10 +13,8 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 )
 
-type pathsState = PathsState
-
 type State struct {
-	*pathsState
```

Comment:
`State` obsahoval `PathsState` cez `embedding`.
Note: pouzival sa alias hore `type pathsState = PathsState` -> male pismeno v nazve, hodnota sa teda nedala mimo balicka prepisat, ... no `State` mal vsetky metody z `PathsState`.

Nevyhoda je, ze sa da z ineho balicka zavolat `MarkTracked` ... a to nechceme.
Chceme aby sa dal iba citat stav `paths`, a modifikovat sa bude iba interne.